.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: ['18', '20']
```

Comment:
Can consider moving up to 22 at some point as it looks like 18 and 20 are in [maintainance mode](https://nodejs.org/en/about/previous-releases)